tests/system/src/test/java/test/javafx/stage/RestoreStagePositionTest.java

```diff
@@ -130,7 +127,7 @@ public void testDemaximizedPosition() throws Exception {
         double y = stage.getY();
 
         Platform.runLater(() -> stage.setMaximized(true));
-        Thread.sleep(200);
+        Thread.sleep(800);
```

Comment:
Would the `test.util.Util.waitForIdle()` be of use here?  Or `runAndWait()` ?

Reply:
Yes on the runAndWait (always a good idea in cases like this where you aren't otherwise latching), but no on the change to use waitForIdle (see my other comment).